doc/source/operations.rst

```diff
@@ -55,23 +55,60 @@ Once the appropriate profile has been selected, it can be applied by running the
+
+Updating OpenStack Version
+--------------------------
+
+To make a global change to the OpenStack version used by all component images, create a key in ${WORKSPACE}/env/extravars called "suse_osh_image_version" and set it to the desired value. For example, to use the "ocata" version, add the following line to the extravars file:
+
+.. code-block:: yaml
+
+   suse_osh_image_version: "ocata"
+
+It is also possible to update an individual image or subset of images to a different version, rather than making a global change. To do this, it will be necessary to manually edit the versions.yaml file located in socok8s/site/soc/software/config/. Locate the images that need to be changed in the "images" section of the file and modify the line to include the desired version. For example, to use the "rocky" version for the heat_api image, change the following line in versions.yaml from
+
+.. code-block:: yaml
+
+   heat_api: "{{ suse_osh_registry_location }}/openstackhelm/heat:{{ suse_openstack_image_version }}"
+
+to
+
+.. code-block:: yaml
 
-Update OpenStack Services
--------------------------
+   heat_api: "{{ suse_osh_registry_location }}/openstackhelm/heat:rocky"
 
-Update secrets, passwords and certificates
+Updating Individual Images and Helm Charts
 ------------------------------------------
 
-Update this repository
-----------------------
+The versions.yaml file can also be used for more advanced update configurations such as using a specific image or Helm chart source version. 
 
-The SUSE Containerized OpenStack repository can be updated by performing the git pull command from the socok8s directory:
+.. note::
+   
+   The following configuration options are not supported for the SUSE Containerized OpenStack Technical Preview. Please proceed at your own risk.
 
-.. code-block:: console
+To specify the use of an updated or customized image, locate the appropriate image name in socok8s/site/soc/software/config/versions.yaml and modify the line to include the desired image location and tag. For example, to use a new heat_api image, modify its entry with the new image location:
+
+.. code-block:: yaml
+
+   heat_api: "registry_location/image_directory/image_name:tag"
+
+Similarly, the versions.yaml file can be used to retrieve a specific version of any Helm chart being deployed. To do so, it will be necessary to provide a repository location, type, and a reference. For example, to use a specific version of the Helm chart for Heat, add the following information to the "osh" section under "charts":
+
+.. code-block:: yaml
+
+     heat:
+       location: https://git.openstack.org/openstack/openstack-helm
+       reference: ${COMMIT_ID}
```

Comment:
can reference be a branch? like `0.1.0` ? if upstream starts using version numbers that may be a much nicer friendly than a commit_id which some end users may not even know how to find

Reply:
Ah, good catch. Yes, the 'reference' key can be a branch, commit, or reference and it defaults to master if not specified. We've been using commit IDs during development, so I must have been on auto-pilot when I wrote this.